services/ui-src/src/styles/_header.scss

```
@@ -101,61 +99,62 @@
 
       &.open {
         background: $brand-tertiary;
-        display: block;
-        padding: ($padding * 2);
+        display: flex;
+        flex-direction: column;
+        align-items: baseline;
+        padding: $padding;
         position: absolute;
         right: 0;
         transition: all 2s;
       }
-
-      .manage-account {
-        padding: $padding 0 $padding ($padding * 4);
+      .contact-us {
```

Same as the suggestion above re: tabbed focus styling.
```suggestion
      a, button {
        &:focus {
          background-color: initial;
        }
      }
      .contact-us {
```